src/common/commonutils/FileUtils.c

```diff
@@ -1160,23 +1155,25 @@ int CheckMarkedTextNotFoundInFile(const char* fileName, const char* text, const
                 }
                 else if (0 == isalpha(found[0]))
                 {
-                    OsConfigLogInfo(log, "CheckMarkedTextNotFoundInFile: '%s' containing '%s' found in '%s' ('%s')", text, marker, fileName, found);
-                    OsConfigCaptureReason(reason, "'%s' containing '%s' found in '%s' ('%s')", text, marker, fileName, found);
+                    OsConfigLogInfo(log, "CheckMarkedTextNotFoundInFile: '%s' containing '%s' found in '%s' uncommented with '%c'", text, marker, fileName, commentCharacter);

```

Comment:
shouldn't it say `... uncomment by removing '...'` and not `... uncomment with '...'`

Reply:
It's 'uncommented'.  Blah found in file foo, Uncommented with '#', for example.

Reply:
i see, thanks for clarifying, makes sense